client/js/InitVue.js

```diff
@@ -24,6 +25,7 @@ import { initStore } from '@DpJs/store/core/initStore'
 import { loadLibs } from '@DpJs/lib/core/loadLibs'
 import loadSentry from './loadSentry'
 import NotificationStoreAdapter from '@DpJs/store/core/NotificationStoreAdapter'
+import NavButton from './components/button/NavButton'
```

Comment:
I'm not sure that the component name represents its purpose. I think this is more likely a `ToggleMenuButton` from my perspective. However `NavButton` kind of suggests that the button is used to navigate but actually it toggles the navigation menu.

Reply:
i took `HamburgerMenuButton` as name. done here https://github.com/demos-europe/demosplan-core/pull/1389/commits/692686822ba44f7c25e4e7bb57ddbbcf42bcf6af